bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/NativeInterface.cs

```diff
@@ -150,6 +150,71 @@ internal static unsafe class NativeInterface
 
         internal unsafe struct CharPtr { };
 
+        #region Collections
+
+        internal static unsafe class LinkedList
+        {
+            internal unsafe struct LinkedListType { };
```

Comment:
we use an opaque struct as the base type (which just resolves to a pointer instance)